src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -91,8 +94,8 @@ export const TxDataRow = ({
           <Value method={method} type={paramType} value={value} />
         </ValueWrapper>
       )}
-      {generateInlineTypeValue(inlineType, value, hasExplorer)}
-      {!inlineType && !isArray && value && (
+      {!isArray && generateInlineTypeValue(inlineType, value, hasExplorer)}
```

Comment:
Can this not work with arrays?

Reply:
I removed the `!isArray` condition from https://github.com/gnosis/safe-react/pull/3600/files#diff-36f6da0c3553940acb58925da0279d1d7f2235a584f5dc05172d99405c24461bL27-R28 but then had to add the condition here